webapp/src/ts/modules/tasks/tasks-content.component.ts

```diff
@@ -308,13 +326,14 @@ export class TasksContentComponent implements OnInit, OnDestroy, AfterViewInit {
         console.debug('saved report and associated docs', docs);
         this.globalActions.setSnackbarContent(this.translateService.instant('report.created'));
 
+        this.tasksActions.setLastCompletedTask(this.selectedTask);
```

Comment:
We don't, unfortunately. We know that the selected task's action was executed. We don't know what effect it will have, once the tasks are recalculated. 
All we get out of the rules engine is the fresh set of tasks, without knowing which action effected which change.

Reply:
so `this.tasksActions.setLastCompletedTask` might not be actually completed? Maybe we can named it something different to not confuse the actual state of task 🤔

Reply:
`setLastSubmittedTask` ?

Reply:
yep!